src/jdk.compiler/share/classes/com/sun/tools/javac/tree/DCTree.java

```diff
@@ -53,41 +84,150 @@
 public abstract class DCTree implements DocTree {
 
     /**
-     * The position in the comment string.
-     * Use {@link #getSourcePosition getSourcePosition} to convert
+     * The position of the first character that is unique to this node.
+     * It is normally set by the methods in {@link DocTreeMaker}.
```

Comment:
Since this change also includes cleanup, we might as well fix "the the" and "non-white space" at DocTreeMaker.java:699.

Reply:
Noted